CHANGELOG.md

```diff
@@ -3,20 +3,17 @@ Version 0.7.0
 
 **New features**
 
-- Add action group `apps` to tool `app-store-connect`.
-- Add action group `beta-app-review-submissions` to tool `app-store-connect`.
-- Add action group `builds` to tool `app-store-connect`.
-- Add action `app-store-connect apps get` to ...
-- Add action `app-store-connect apps list` to ...
-- Add action `app-store-connect apps app-store-versions` to ...
-- Add action `app-store-connect apps builds` ...
-- Add action `app-store-connect apps pre-release-versions` ...
-- Add action `app-store-connect beta-app-review-submissions create` ...
-- Add action `app-store-connect beta-app-review-submissions list` ...
-- Add action `app-store-connect builds pre-release-version` to ...
-- Add action `app-store-connect publish` to ...
-- Add action `xcode-project ipa-info` to ...
-- Add action `xcode-project pkg-info` to ...
+- Add action `app-store-connect apps get` to get information about a specific app.
+- Add action `app-store-connect apps list` to find apps added in App Store Connect.
+- Add action `app-store-connect apps app-store-versions` to find App Store versions associated with a specific app.
+- Add action `app-store-connect apps builds` to find builds associated with a specific app.
+- Add action `app-store-connect apps pre-release-versions` to find prerelease versions associated with a specific app.
+- Add action `app-store-connect beta-app-review-submissions create` to submit an app for beta app review to allow external testing.
+- Add action `app-store-connect beta-app-review-submissions list` to find beta app review submissions of a build.
+- Add action `app-store-connect builds pre-release-version` to find the prerelease version for a specific build
+- Add action `app-store-connect publish` to upload application packages to App Store and submit them to Testflight.
```

Comment:
Why is this `publish` and not `testflight publish` or `publish testflight` I ask because what if we add automatic app submissions? Would this be in another command, not app-store-connect?

Reply:
It is named `publish` because submitting to TestFlight is optional. Also, later down the line we can add option to submit the published version directly to App Store.